plugins/uefi/fu-plugin-uefi.c

```diff
@@ -582,6 +595,30 @@ fu_plugin_uefi_test_secure_boot (FuPlugin *plugin)
 	fu_plugin_add_report_metadata (plugin, "SecureBoot", result_str);
 }
 
+static gboolean load_custom_esp (FuPlugin *plugin, GError **error)
+{
+	FuPluginData *data = fu_plugin_get_data (plugin);
+	const gchar *key = "OverrideESPMountPoint";
+
+	data->esp_path = fu_plugin_get_setting (plugin, key);
+	if (data->esp_path) {
+		if (!g_file_test (data->esp_path, G_FILE_TEST_IS_DIR)) {
+			g_set_error (error,
+				     FWUPD_ERROR,
+				     FWUPD_ERROR_INVALID_FILE,
+				     "Invalid %s specified: %s",
```

Comment:
Maybe also specify where the "invalid" value was set?

Reply:
As in whole path?  or just "UEFI conf" file?  Sorta gets rid of abstraction if the plugin knows about the whole path here.

Reply:
True, maybe leave that for now.

Reply:
I just changed it to refer to `"%s config", fu_plugin_get_name (plugin)` which will keep it abstracted but still point them where to fix it.